### PR TITLE
User view

### PR DESCRIPTION
**Pull request by fgerschwiler**
Merge after user command and project query

### CODE REVIEW

File: internal/user/repository/view/user_view.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	}
+	query = view.PrepareGetByKey(table, model.UserSearchKey(usr_model.USERSEARCHKEY_USER_NAME), email)
+	err = query(db, user)
+	if err != nil && !caos_errs.IsNotFound(err) {
+		return false, err
+	}
+	if user != nil {
+		return false, nil
+	}
+	return true, nil
```
Comment (adlerhurst) [inline internal/user/repository/view/user_view.go:60]:
```suggestion
	return user == nil, nil
```

File: pkg/management/api/grpc/user_converter.go [hunk 2]
```diff
@@ -207,6 +252,76 @@ func updateAddressToModel(address *UpdateUserAddressRequest) *usr_model.Address
...
+		Limit:       response.Limit,
+		TotalResult: response.TotalResult,
+		Result:      userViewsFromModel(response.Result),
+	}
+}
+
+func userViewsFromModel(users []*usr_model.UserView) []*UserView {
+	converted := make([]*UserView, len(users))
+	for i, q := range users {
+		converted[i] = userViewFromModel(q)
```
Comment (adlerhurst) [inline pkg/management/api/grpc/user_converter.go:267]:
```suggestion
	for i, user := range users {
		converted[i] = userViewFromModel(user)
```

File: pkg/management/api/grpc/user_grant_converter.go [hunk 3]
```diff
@@ -64,6 +64,85 @@ func projectGrantUserGrantUpdateToModel(u *ProjectGrantUserGrantUpdate) *grant_m
...
+		Limit:       response.Limit,
+		TotalResult: response.TotalResult,
+		Result:      userGrantViewsFromModel(response.Result),
+	}
+}
+
+func userGrantViewsFromModel(users []*grant_model.UserGrantView) []*UserGrantView {
+	converted := make([]*UserGrantView, len(users))
+	for i, q := range users {
+		converted[i] = userGrantViewFromModel(q)
```
Comment (adlerhurst) [inline pkg/management/api/grpc/user_grant_converter.go:116]:
```suggestion
	for i, user := range users {
		converted[i] = userGrantViewFromModel(user)
```

File: internal/user/model/user_view.go [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+func (r *UserSearchRequest) EnsureLimit(limit uint64) {
+	if r.Limit == 0 || r.Limit > limit {
+		r.Limit = limit
+	}
+}
+
+func (r *UserSearchRequest) AppendMyOrgQuery(ctx context.Context) {
+	orgID := grpc_util.GetHeader(ctx, api.ZitadelOrgID)
```
[the remark this answers is not in the document]
Comment (fgerschwiler) [reply internal/user/model/user_view.go:82]: changed it every where

**Pull request merged**